aries-site/src/themes/aries.js

```diff
@@ -6,10 +6,42 @@ export const aries = deepMerge(hpe, {
   // To be stripped out once theme changes are made in grommet-theme-hpe
   // keeping file for use as playground for future theme adjusments that need
   // to be quickly tested
-  menu: {
-    group: {
-      separator: {
-        color: 'border-weak',
+  feedback: {
+    body: {
+      gap: 'medium',
+    },
+    button: {
+      a11yTitle: `You are in a layer containing a form to submit feedback.
+        To close this layer, press Enter.`,
+    },
+    container: {
+      pad: 'medium',
+    },
+    header: {
```

Comment:
should we have the heading props listed under `feedback.heading`?

For example:
```
feedback: { 
   heading: {
   // any heading props
   }
   header: {
   // any box props
   }
}
```
Then we can eliminate the `container` stuff